docs/reference/method/MongoDBModelIndexInfo-is2dSphere.txt

```diff
@@ -0,0 +1,58 @@
+==========================================
+MongoDB\\Model\\IndexInfo::is2dSphere()
+==========================================
+
+.. versionadded:: 1.4
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Model\\IndexInfo::is2dSphere()
+
+   Return whether or not the index is a :manual:`2dsphere </core/2dsphere>`
```

Comment:
I think we can remove "or not" to be consistent with other documentation for "is" methods.